ethsigner/src/main/java/tech/pegasys/ethsigner/requesthandler/sendtransaction/SendTransactionHandler.java

```diff
@@ -71,18 +73,18 @@ public void handle(final RoutingContext context, final JsonRpcRequest request) {
       return;
     }
 
-    if (senderNotUnlockedAccount(params)) {
+    if (senderNotUnlockedAccount(transaction.sender())) {
```

Comment:
nit; As `senderNotUnlockedAccount()` is a private method, why not pass in `transaction` object instead of the String?
_(cleaner looking code)_

Reply:
good idea. done